buildkite/resource_test_suite.go

```diff
@@ -54,19 +55,45 @@ func (ts *testSuiteResource) Create(ctx context.Context, req resource.CreateRequ
 	payload := map[string]interface{}{}
 	var teamOwnerUuid string
 
-	resp.Diagnostics.Append(req.Plan.Get(ctx, &plan)...)
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+
 	if resp.Diagnostics.HasError() {
 		return
 	}
 
+	timeout, diags := ts.client.timeouts.Read(ctx, DefaultTimeout)
```

Comment:
Hmm I think this should still use the create timeout value. Or at least the read part of this create function use the read timeout and the create part of it use the create part

Reply:
Just whipped that in wit the latest commit - `POST` to test suite creation endpoint using the `Create` timeout - above with Team owner UUID -> ID using the `Read` T/O